third_party/nvfuser/csrc/transform_iter.h

```diff
@@ -39,80 +39,98 @@ struct id_int_lt {
 // If error_on_failure = false, replay will replay everything it can, and ignore
 // operations it can't.
 class TORCH_CUDA_CU_API ReplayTransformations : public IterVisitor {
- protected:
-  // NOLINTNEXTLINE(cppcoreguidelines-non-private-member-variables-in-classes)
```

Comment:
I didn't like this `NOLINTNEXTLINE` annotation, so I moved them to `private` when easily done. Still some have the annotation, but looks cleaner than before.